cmake/init.cmake

```diff
@@ -162,15 +162,15 @@ macro(prepare_build)
     check_include_files (mysql/mysql.h HAVE_MYSQL)
     check_include_files (zlib.h HAVE_ZLIB)
   endif()
-  if (NOT HAVE_OPENSSL)
-    message(FATAL_ERROR "OpenSSL not found")
-  endif()
-  if (NOT HAVE_ZLIB)
-    message(FATAL_ERROR "ZLib not found")
-  endif()
-  if (NOT HAVE_MYSQL)
-    message(WARNING "MySQL not found")
-  endif()
+  #if (NOT HAVE_OPENSSL)
+  #  message(FATAL_ERROR "OpenSSL not found")
+  #endif()
+  #if (NOT HAVE_ZLIB)
+  #  message(FATAL_ERROR "ZLib not found")
+  #endif()
+  #if (NOT HAVE_MYSQL)
+  #  message(WARNING "MySQL not found")
+  #endif()
```

Comment:
This is temporary, only because I could not figure out how to get this to work with CLion on OSX